### PR TITLE
Bump site dependency

### PR DESCRIPTION
**Pull request by charleswaddell**

### CODE REVIEW

**Comment by gauthierm** [comment]
If you want to support multiple major versions, this should use `^5.0.0 || ^6.0.0`. It will prevent it from breaking when version X is released and is incompatible.

**Comment by charleswaddell** [comment]
> If you want to support multiple major versions, this should use ^5.0.0 || ^6.0.0. It will prevent it from breaking when version X is released and is incompatible.

My reasoning was to prevent this problem going forward. If I used `^5.0.0 || ^10.0.0` then we might run into this again later. When we have Site version `14.0.0`. Maybe that's not worth it but I can't see us these APIs from Site going forward so I figured it would be a safe fix.

**Comment by gauthierm** [comment]
I think it's still better to be more explicit about what major version is supported. Who knows what we'll decide to do with the APIs in 2-3 years. (you're right that they probably won't change)

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "require": {
     "php": ">=5.3.0",
     "psr/log": "^1.0.0",
-    "silverorange/site": "^5.0.0"
+    "silverorange/site": ">=5.0.0"
```
Comment (charleswaddell) [inline composer.json:26]:
```suggestion
    "silverorange/site": "^5.0.0 || ^10.0.0"
```

**Pull request merged**